src/master/flags.hpp

```diff
@@ -111,6 +111,7 @@ class Flags : public virtual logging::Flags
   // Optional IP discover script that will set the Master IP.
   // If set, its output is expected to be a valid parseable IP string.
   Option<std::string> ip_discovery_command;
+  bool enable_network_bandwidth_enforcement;
```

Comment:
not important but line placement is a bit wierd, seems to create a paragraph with the line of code/comments above...